Sources/JExtractSwiftLib/JNI/JNISwift2JavaGenerator+JavaTranslation.swift

```diff
@@ -253,6 +274,70 @@ extension JNISwift2JavaGenerator {
       }
     }
 
+    func translateOptionalParameter(
+      wrappedType swiftType: SwiftType,
+      parameterName: String
+    ) throws -> TranslatedParameter {
+      let parameterAnnotations: [JavaAnnotation] = getTypeAnnotations(swiftType: swiftType, config: config)
+
+      switch swiftType {
+      case .nominal(let nominalType):
+        let nominalTypeName = nominalType.nominalTypeDecl.name
+
+        if let knownType = nominalType.nominalTypeDecl.knownTypeKind {
+          guard let javaType = JNIJavaTypeTranslator.translate(knownType: knownType, config: self.config) else {
+            throw JavaTranslationError.unsupportedSwiftType(swiftType)
+          }
+
+          let (translatedClass, orElseValue) = switch javaType {
+          case .boolean: ("Optional<Boolean>", "false")
+          case .byte: ("Optional<Byte>", "(byte) 0")
+          case .char: ("Optional<Character>", "(char) 0")
+          case .short: ("Optional<Short>", "(short) 0")
+          case .int: ("OptionalInt", "0")
+          case .long: ("OptionalLong", "0L")
+          case .float: ("Optional<Float>", "0f")
+          case .double: ("OptionalDouble", "0.0")
+          case .javaLangString: ("Optional<String>", #""""#)
```

Comment:
```suggestion
          case .javaLangString: ("Optional<String>", "null")
```

can we use null for _all_ other values that are not primitives? We'd remove the default's throw here as well and just default everything to null

Reply:
yup, good idea